test/units/galaxy/test_api.py

```diff
@@ -75,6 +75,60 @@ def get_test_galaxy_api(url, version, token_ins=None, token_value=None):
     return api
 
 
+def get_collection_versions(namespace='namespace', name='collection'):
+    base_url = 'https://galaxy.server.com/api/v2/collections/{0}/{1}/'.format(namespace, name)
+    versions_url = versions_url = base_url + 'versions/'
```

Comment:
Typo?
```suggestion
    versions_url = base_url + 'versions/'
```

Reply:
Heh, I thought I deleted that once before